database/forms.py

```diff
@@ -11,6 +11,7 @@
 from database.models.structural_models import ModelSet, SetMembership
 from database.models.user_models import Teacher, Student
 
+
```

Comment:
reformat file with the pycharm reformat code feature to eliminate spacing issues

Reply:
will do